backend/manager/crawler.py

```diff
@@ -350,20 +350,24 @@ def parse(year, semester, lectures, field_name):
                                                     name=lecture['name'],
                                                     type=lecture['type'],
                                                     field=field_name,
-                                                    grade=int(lecture['grade'][0]) if not lecture['grade'] else 0,
+                                                    grade=int(lecture['grade'][0]) if len(lecture['grade']) != 0 else 0,
```

Comment:
`len(lecture['grade']) != 0` is equal to `lecture['grade']`, in this case.

Reply:
Sure. Why not.